cmd/dm-pocket/main.go

```diff
@@ -32,6 +34,11 @@ var (
 
 func main() {
 	flag.Parse()
+
+	go func() {
```

Comment:
Tipocket opens the pprof by default so you don't need to open it again.

https://github.com/pingcap/tipocket/blob/35dea2cc300e17a19aebbba537d5e01a5e390cc8/pkg/test-infra/fixture/fixture.go#L277-L279

Reply:
Oh, thanks a lot, removed in https://github.com/pingcap/tipocket/pull/258/commits/473df6662695c10bd3934b83cce7b83b65e6d02d.

BTW, mirror update in https://github.com/pingcap/tipocket/pull/258/commits/d5336e7e5d410f87a403111f285e329113d936bc.